pkg/k8sutil/k8sutil.go

```diff
@@ -179,6 +179,6 @@ func isKubeMetaKind(kind string) bool {
 	return false
 }
 
-func isRunModeLocal() bool {
+func IsRunModeLocal() bool {
```

Comment:
This change to export `isRunModeLocal` should be able to be reverted since we shouldn't be checking this in the watches.yaml loader code.

Reply:
See the comment above.